data/strings/types_strings.txt

```diff
@@ -9193,6 +9193,318 @@
+    hu = Podiatrist
+    id = Ahli penyakit kaki
+    it = Podologo
+    ja = 足病医
+    ko = 발 전문의
+    mr = पोडियाट्रिस्ट
+    nb = Fotterapeut
+    nl = Podotherapeut
+    pl = Podolog
+    pt = Podiatrista
+    pt-BR = Podiatrista
+    ro = Podiatru
+    ru = Врач-ортопед
+    sk = Podológ
+    sv = Fotvårdare
+    sw = Daktari wa miguu
+    th = แพทย์โรคเท้า
+    tr = Podiatrist
+    uk = Лікар-ортопед
+    vi = Bác sĩ chân
+    zh-Hans = 足科医生
+    zh-Hant = 足病醫生
+
+  [type.healthcare.psychotherapist]
+    en = Psychotherapist
+    ar = معالج نفسي
+    be = Псіхатэрапеўт
+    bg = Психотерапевт
+    cs = Psychoterapeut
+    da = Psykoterapeut
+    de = Psychotherapeutin
+    el = Ψυχοθεραπευτής
+    es = Psicoterapeuta
+    et = Psühhoterapeut
+    eu = Psikoterapeuta
+    fa = روان درمانگر
+    fi = Psykoterapeutti
+    fr = Psychothérapeute
+    he = פסיכותרפיסט
+    hu = Pszichoterapeuta
+    id = Psikoterapis
+    it = Psicoterapeuta
+    ja = サイコセラピスト
+    ko = 심리 치료사
+    mr = मानसोपचारतज्ज्ञ
+    nb = Psykoterapeut
+    nl = Psychotherapeut
+    pl = Psychoterapeuta
+    pt = Psicoterapeuta
+    pt-BR = Psicoterapeuta
+    ro = Psihoterapeut
+    ru = Психотерапевт
+    sk = Psychoterapeut
+    sv = Psykoterapeut
+    sw = Mwanasaikolojia
+    th = นักจิตบำบัด
+    tr = Psikoterapist
+    uk = Психотерапевт
+    vi = nhà trị liệu tâm lý
+    zh-Hans = 心理治疗师
+    zh-Hant = 心理治療師
+
+  [type.healthcare.sample_collection]
+    en = Sample collection
+    ar = جمع العينات
+    be = Калекцыя ўзораў
+    bg = Събиране на проби
+    cs = Sběr vzorků
+    da = Prøveudtagning
+    de = Probenahme
+    el = Συλλογή δείγματος
+    es = Recogida de muestras
+    et = Proovide kogumine
+    eu = Lagin-bilketa
+    fa = مجموعه نمونه
+    fi = Näytteenotto
+    fr = Collecte d'échantillons
+    he = אוסף דוגמאות
+    hu = Mintavétel
+    id = Pengumpulan sampel
+    it = Raccolta di campioni
+    ja = サンプル採取
+    ko = 샘플 수집
+    mr = नमुना संकलन
+    nb = Prøvesamling
+    nl = Monstername
+    pl = Zbieranie próbek
+    pt = Recolha de amostras
+    pt-BR = Coleta de amostras
+    ro = Colectarea probelor
+    ru = Сбор образцов
+    sk = Zber vzoriek
+    sv = Insamling av prover
+    sw = Mkusanyiko wa sampuli
+    th = เก็บตัวอย่าง
+    tr = Örnek toplama
+    uk = Відбір проб
+    vi = Bộ sưu tập mẫu
+    zh-Hans = 样品收集
+    zh-Hant = 樣品採集
+
+  [type.healthcare.speech_therapist]
+    en = Speech therapist
+    ar = معالج النطق
+    be = Лагапед
+    bg = Логопед
+    cs = Logoped
+    da = Talepædagog
+    de = Logopädin
+    el = Λογοθεραπευτής
+    es = Logopeda
+    et = Logopeed
+    eu = Logopeda
+    fa = متخصص گفتار درمانی
+    fi = Puheterapeutti
+    fr = Orthophoniste
+    he = קלינאי תקשורת
+    hu = Logopédus
+    id = Terapis bicara
+    it = Logopedista
+    ja = スピーチセラピスト
+    ko = 언어 치료사
+    mr = स्पीच थेरपिस्ट
+    nb = Taleterapeut
+    nl = Logopedist
+    pl = Logopeda
+    pt = Fonoaudióloga
+    pt-BR = Fonoaudióloga
```

Comment:
pt = Audiologista
pt-BR = Fonoaudiólogo

Reply:
Thanks!
Done in 5d32dd98de7b8591e4719e2842b617d50026572f